.golangci.yml

```diff
@@ -19,11 +19,11 @@ linters-settings:
 linters:
   disable-all: true
   enable:
-    - errcheck
-    - ineffassign
-    - gas
+    #- errcheck
```

Comment:
Is this disabling them or enabling them? I am confused.

Reply:
as part of this PR https://github.com/dgraph-io/dgraph/pull/8620/files#diff-6179837f7df53a6f05c522b6b7bb566d484d5465d9894fb04910dd08bb40dcc9L22-R31 .. we tried addressing all the linters by uncommenting it. 

The rebase issue hit us in that PR. The linter ran fine on that particular PR, but failed after a merge to `main`. This PR is partially fixing errors from 2 linters & taking things back to what it was before.

Reply:
Understood.